lessons/operator/03/README.md

```diff
@@ -375,208 +217,446 @@ $ sensuctl configure -n \
+   
+   1. Modify the `.envrc` or `.envrc.ps1`, replacing the value for `SENSU_API_KEY` with the key we just copied, then uncomment the line.
+   
+      When complete, your file should have a line like this:
 
-   ```shell
-   sensuctl event info learn.sensu.io helloworld --format json
-   ```
+      **Mac and Linux users (`.envrc`):**
 
-   > **NOTE:** the `--output` flag is available for a variety of `sensuctl` commands, which can be quite helpful for writing scripts that leverage data from Sensu APIs.
+      ```shell
+      export SENSU_API_KEY=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+      ```
 
-**NEXT:** If `sensuctl` output a JSON formatted event, you're ready to move on to the next step.
+      **Windows users (`.envrc.ps1`):**
 
-## EXERCISE 5: explore Sensu resources using `sensuctl`
+      ```powershell
+      ${Env:SENSU_API_KEY}="xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+      ```
+   
+1. **Reload the Environment Variables.**
 
-1. **Use the `sensuctl namespace list` command to get a list of namespaces.**
+   **Mac and Linux:**
 
    ```shell
-   sensuctl namespace list
+   source .envrc
    ```
 
-   _NOTE: the output of this command is filtered based on RBAC, so different users may see different results._
+   **Windows (Powershell):**
 
-1. **Use the `sensuctl event list` command to get a list of events.**
+   ```powershell
+   . .\.envrc.ps1
+   ```
 
+1. Verify that the `SENSU_API_KEY` environment variable is set:
+
+   **Mac and Linux:**
+ 
    ```shell
-   sensuctl event list
+   echo $SENSU_API_KEY
    ```
 
-1. **Use the `sensuctl entity list` command to get a list of nodes under management.**
+   **Windows (Powershell):**
 
-   ```shell
-   sensuctl entity list
+   ```powershell
+   $env:SENSU_API_KEY
    ```
 
-   _NOTE: try adding `--format json` or `--format yaml` to view the list in JSON or YAML format._
+> **PROTIP:** If you like to automate things like this using shell scripts, you might already be thinking about how to parse the output of the `sensuctl api-key grant` command.
+>
+> The following example should do the trick for Mac and Linux users:
+>
+> **Example:** Parsing an API key using `awk`
+> ```shell
+> export SENSU_API_KEY=$(sensuctl api-key grant sensu | awk -F "/" '{print $NF}')
+> ```
 
-1. **Get information about a specific entity using the `sensuctl entity info` command.**
+## Listing and Viewing Resources
 
-   ```shell
-   sensuctl entity info learn.sensu.io
-   ```
+One of the common uses of `sensuctl` is to list and view resources like namespaces, users, and entities. 
 
-1. **Try exploring some other resources.**
+### EXERCISE 4: List Sensu Resources
+#### Scenario
 
-   _NOTE: don't forget to use `--help`; for example, `sensuctl --help` will output a list of "management commands" which are effectively API resources that are accessible via `sensuctl`._
+You want to explore the resources available in Sensu. 
```

Comment:
I don't love the phrase "under management by" because it's passively voiced and wordy. I went ahead and replaced it because I do see the distinction. 

I am not sure I completely agree that it's worth replacing the text for that though. "Available" here doesn't imply, to me, that it is a *complete* list of *possible* available things, but rather the things that are *currently available*/actionable/active/that Sensu is currently aware of/etc. Availability in a dist-sys context (what service nodes are available right now?) is a constantly changing thing, and implies that they are usable at the moment. Availability in the way you're using it here is more of a sale-mentality "We have these 5 models available from our supplier, though some might need to be special-ordered. What's in-stock today are these two models."...

Maybe we can brainstorm a phrase that means the same as my version of "available" without being too wordy or passive or implying other meanings.

Reply:
Opened an issue for follow up here: https://github.com/sensu/sensu-go-workshop/issues/76